docs/index.restdown

```diff
@@ -1081,6 +1082,21 @@ Options:
 
 Note that `ip`, `xff` and `username` are XOR'd.
 
+### Request Expiry
+
+Request expiry is used to kick out old requests that have been timed out somewhere upstream.
+The plugin will take in the key to use for looking up the expired time in the header.  The
+expired time should be absolute time.  If the expire time is in the future (greater than
+`Date.now()` or does not exist the request will continue to execute the chain.  If the
+provided expiry time is less than `Date.now()` next will be provided a 504, Gateway
+Timeout Error.
+
+    server.use(restify.requestExpiry({
+        header: 'x-request-expiry-time'
+    });
+
+The only options provided is `header` which is the key used to look in each request header.
```

Comment:
The only options provided is `header` which is the request header used to look specify the client timeout.